src/core/ee/ipu/ipu.cpp

```diff
@@ -1014,6 +1041,113 @@ bool ImageProcessingUnit::process_CSC()
     }
 }
 
+bool ImageProcessingUnit::process_PACK()
+{
+    while (true)
+    {
+        switch (pack.state)
+        {
+        case PACK_STATE::BEGIN:
```

Comment:
Add an extra layer of indentation to all case statements in a switch block.

Reply:
Indented.